src/main/resources/native/ios/JvmFuncsFallbacks.c

```diff
@@ -28,11 +28,6 @@ JNIEXPORT jobject JNICALL JVM_AreNestMates(JNIEnv *env) {
     return NULL;
 }
 
-JNIEXPORT jobject JNICALL JVM_ArrayCopy(JNIEnv *env) {
```

Comment:
As a follow-up, this whole file should be removed, as it is obsoleted by libvmone.

Reply:
I take that all these dummy functions are not used at all, but are not in vmone yet (otherwise we'd get duplicated symbols). Do you mean we should move them to vmone?